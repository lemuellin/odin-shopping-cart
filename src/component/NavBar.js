import React from 'react';
import { Link } from 'react-router-dom';
import '../style/nav.css';

const NavBar = () => {
    return(
        <nav>
            <h1 className='navLogo'>Minimal Garden</h1>
            <ul className='navLinkContainer'>
                <Link to='/shopping-cart' className='navLink' name='Home'>
                    <li>Home</li>
                </Link>
                <Link to='/shop' className='navLink'>
                    <li>Shop</li>
                </Link>
                <Link to='/contact' className='navLink'>
                    <li>Contact</li>
                </Link>
            </ul>
        </nav>
    )
}

export default NavBar;