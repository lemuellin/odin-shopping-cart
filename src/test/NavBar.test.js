import React from "react";
import { render } from "@testing-library/react";
import { BrowserRouter } from 'react-router-dom';
import NavBar from "../component/NavBar.js";


describe("NavBar", () => {
    it("Match Snapshot", () => {
        const { container } = render (
            <BrowserRouter>
                <NavBar />
            </BrowserRouter>
        );
        expect(container).toMatchSnapshot();
    });

});