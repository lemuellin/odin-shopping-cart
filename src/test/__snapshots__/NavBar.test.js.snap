// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar Match Snapshot 1`] = `
<div>
  <nav>
    <h1
      class="navLogo"
    >
      Minimal Garden
    </h1>
    <ul
      class="navLinkContainer"
    >
      <a
        class="navLink"
        href="/shopping-cart"
        name="Home"
      >
        <li>
          Home
        </li>
      </a>
      <a
        class="navLink"
        href="/shop"
      >
        <li>
          Shop
        </li>
      </a>
      <a
        class="navLink"
        href="/contact"
      >
        <li>
          Contact
        </li>
      </a>
    </ul>
  </nav>
</div>
`;

exports[`NavBar can switch to Home route 1`] = `
<div>
  <nav>
    <h1
      class="navLogo"
    >
      Minimal Garden
    </h1>
    <ul
      class="navLinkContainer"
    >
      <a
        class="navLink"
        href="/shopping-cart"
        name="Home"
      >
        <li>
          Home
        </li>
      </a>
      <a
        class="navLink"
        href="/shop"
      >
        <li>
          Shop
        </li>
      </a>
      <a
        class="navLink"
        href="/contact"
      >
        <li>
          Contact
        </li>
      </a>
    </ul>
  </nav>
</div>
`;
